### DISCUSSION TITLE
Error code NO_RESPONSE_FROM_FUNCTION after deployment

### DISCUSSION THREAD

**Discussion by manhthepixta** [vercel/vercel#4786] [Help] [answered]
I got this error after deployment:

```
An error occurred with this application.
This is an error with the application itself, not the platform.

502: BAD_GATEWAY
Code: NO_RESPONSE_FROM_FUNCTION
ID: hkg1::s7tnj-1594001532301-99963dcad665
```

I wrote this project with next.js and used custom Express server for next-i18next.
Maybe I missed something.
Could you explain why the error occured?

P/s: I used graphql to get data. The endpoint is: `https://domain.com/api`. Does it lead to the error?

**Comment by mcsdevv** [comment, marked as answer]
Hello @manhthepixta, the Vercel platform does not support the use of a Next.js custom server: https://vercel.com/knowledge/npm-run-start-not-working

You should take a look at using Serverless Functions instead: https://vercel.com/docs/v2/serverless-functions/introduction
